lib/node_modules/@stdlib/math/base/special/nonfibonacci/examples/c/example.c

```diff
@@ -0,0 +1,31 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/nonfibonacci.h"
+#include <stdio.h>
+#include <stdint.h>
+
+int main( void ) {
+	int32_t i;
```

Comment:
Shall we use `double i` instead? as we are using `double stdlib_base_nonfibonacci( const double x );` to allow `NaN` as discussed while implementing `fibonacci`.

Reply:
No, we should do as we did in Fibonacci: https://github.com/stdlib-js/stdlib/blob/31632017d48d98b52623397755fe880b08eb1cb4/lib/node_modules/%40stdlib/math/base/special/fibonacci/examples/c/example.c#L24